### PR TITLE
Maven Package clearance code Integration to main

### PR DESCRIPTION
**Pull request by karthika-g**

### CODE REVIEW

File: CA.nuspec
Comment (Vijayalakshmi027) [inline CA.nuspec:1]: image

when i try to generate the nuget package i got this error.. please check

File: TestFiles/IntegrationTestFiles/SystemTest1stIterationData/Maven/POM.xml
Comment (Vijayalakshmi027) [inline TestFiles/IntegrationTestFiles/SystemTest1stIterationData/Maven/POM.xml:1]: run the ut and it in the pipeline and share the results

File: README.md
Comment (Vijayalakshmi027) [inline README.md:1]: image

Please correct it

Comment (karthika-g) [reply README.md:1]: corrected

File: CA.nuspec [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <package >
   <metadata>
     <id>continuous-clearing</id>
-    <version>3.0.0</version>
+    <version>3.0.1</version>
```
Comment (Vijayalakshmi027) [inline CA.nuspec:7]: update the version

File: UsageDoc/CA_UsageDocument.md [hunk 2]
```diff
@@ -239,7 +259,8 @@ Description for the settings in `appSettings.json` file
 | 16    | --artifactoryuploaduser              | Jfrog User Email                              | Yes                                                       |
 | 17  | --jfrognpmdestreponame         | The destination folder name for the NPM package to be copied to                  | Yes                                                    |
```
Comment (Vijayalakshmi027) [inline UsageDoc/CA_UsageDocument.md:260]: Update here

File: src/LCT.Common/appSettings.json
Comment (Vijayalakshmi027) [inline src/LCT.Common/appSettings.json:1]: update version to 3.1.0

File: UsageDoc/CA_UsageDocument.md (hunk 2 above)
Comment (karthika-g) [reply UsageDoc/CA_UsageDocument.md:260]: image is broken

File: CA.nuspec (hunk 1 above)
Comment (karthika-g) [reply CA.nuspec]: done

File: UsageDoc/CA_UsageDocument.md (hunk 2 above)
Comment (karthika-g) [reply UsageDoc/CA_UsageDocument.md:260]: done

File: CA.nuspec [hunk 3]
```diff
@@ -13,7 +13,7 @@
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <description>
       The License clearing tool helps the Project Manager/Developer, to reduce the manual effort and enable the faster license clearing process,
-      by automatically identifying the third party oss components used in their project(i.e., npm, nuget and Debian type) and it creates them in the sw360 and fossology
+      by automatically identifying the third party oss components used in their project(i.e., npm, nuget, maven and Debian type) and it creates them in the sw360 and fossology
       for clearing license
     </description>
     <releaseNotes></releaseNotes>
```
Comment (Vijayalakshmi027) [inline CA.nuspec:19]: add maven in the below tags section

File: TestFiles/IntegrationTestFiles/SystemTest1stIterationData/Maven/POM.xml
Comment (karthika-g) [reply TestFiles/IntegrationTestFiles/SystemTest1stIterationData/Maven/POM.xml:1]: UT and IT passing now 💯  refer pipeline : https://github.com/siemens/continuous-clearing/actions/runs/4635430595/jobs/8202483738

File: CA.nuspec (hunk 3 above)
Comment (karthika-g) [reply CA.nuspec:19]: done

Comment (karthika-g) [reply CA.nuspec:19]: done

File: src/LCT.Common/appSettings.json
Comment (karthika-g) [reply src/LCT.Common/appSettings.json:1]: done

**Pull request merged**